src/routes/index.svelte

```diff
@@ -609,7 +609,7 @@
             </div>
           {:else}
             <div id="iframe-url" class="tile is-child box is-translucent-black has-text-white is-flex-grow-1">
-              <Iframe url={demo?.iframeUrl} />
+              <Iframe url={demo?.iframeUrl} allow={'camera;microphone; fullscreen;display-capture'} />
```

Comment:
We should have a config setting in the builder form to explicitly allow such access.